### PR TITLE
Add podman pause/unpause demo script

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Creates a demo script for pause/unpause.
Touches up podman images script ending slightly.

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@mheon @rhatdan PTAL

File: podman_cli/podman_images.sh [hunk 1]
```diff
@@ -206,7 +206,9 @@ podman_images() {
 
 clean_images_and_containers() {
 
-    read_color "podman rm -a"
+    echo
+    echo_color "Time to clean up!"
+    read_color "podman rm -a -f"
```
Comment (mheon) [inline podman_cli/podman_images.sh]: Is this supposed to be `-a`?

**Comment by mheon** [comment]
One question, otherwise looks fine

**Comment by rhatdan** [comment]
LGTM other then @mheon comment, fix the message or the command.

File: podman_cli/podman_images.sh (hunk 1 above)
Comment (TomSweeneyRedHat) [reply podman_cli/podman_images.sh]: nope, but I neglected to add `-f` to the actual command below it so that it matched.

**Pull request merged**